digiroad2-oracle/sql/insert_municipalities.sql

```diff
@@ -0,0 +1,352 @@
+INSERT INTO municipality (id,name_fi,name_sv,ely_nro,road_maintainer_id,geometry,zoom) VALUES
```

Comment:
tähän varmaan kuuluu taulun manuaalinen tyhjennys? Mistä noi pisteet on napattu?

Reply:
Tää skipti on tehty käyttämällä DBeaver export toimintoa. Exportattu siis lokaali-kehityskannasta alkuperäisen MunicipalityCodeImporter:in CSV-tiedostojen perusteella luoma municipality data, johonka ajettu migraatiot ja  tiketin https://extranet.vayla.fi/jira/browse/DROTH-4164 korjaus skripti. Puuttuvat pisteet ja zoom tasot selittyy sillä, että kunnat_ja_elyt_2018.csv tiedostossa ei ollut rivejä noille parille kunnalle, joten niille ei päivitetty geometry ja zoom sarakkeita

Reply:
Taulu tyhjenee clear-db.sql skriptin ajon yhteydessä kuten muutkin

Reply:
ok